### DISCUSSION TITLE
Introduce avatar

### DISCUSSION THREAD

**Discussion by fabolousfrances** [twbs/bootstrap#38461] [Ideas]
I'd like to see an avatar component in future Bootstrap version.

Which would support icons and images. I think it's an important part of a UI design system.

Thanks 😄

**Comment by patkon** [comment]
You can create your own avatar component with the available resources. And I think that is a lot more flexible than a predefined variant.
Take a look at the examples: https://getbootstrap.com/docs/5.3/examples/offcanvas-navbar/
or https://getbootstrap.com/docs/5.3/examples/sidebars/ (at the bottom of the Page)

**Comment by julien-deramond** [comment]
I agree with @patkon answer. I'm adding this topic to my TODO to check if we can provide somehow in the future a specific example for avatars based on existing utilities and maybe some extra lines of CSS. Even if it probably won't be necessary to create a Bootstrap component (IDK, pure guess here), could be nice to have something ready to copy/paste. There are probably other examples but I like how it is easy for instance to get [avatars in DaisyUI](https://daisyui.com/components/avatar/).

**Comment by fabolousfrances** [comment]
Thanks a lot for your quick answers.
Yes it could be just an example in the doc with existing utilities too instead of a component.
Thanks !
